astropy/time/tests/test_custom_formats.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+
+from astropy.time import Time, TimeFormat
+
+
+def test_custom_time_format_exception():
+    class SpecificException(ValueError):
+        pass
+
+    # Note there is no way to remove this after the test!
+    class Custom(TimeFormat):
+        name = "custom_time_format_test"
+
+        def set_jds(self, val, val2):
+            raise SpecificException
+
+    # with pytest.raises(SpecificException):
```

Comment:
I am guessing `pytest.raises` didn't work? Please remove the commented code if it is not going to be used. Thanks!

Reply:
Or you can try https://stackoverflow.com/questions/38403710/how-can-i-use-pytest-raises-with-multiple-exceptions

Reply:
Yeah, sorry, this was a sort of work-in-progress thing, meant to make the test cope with whatever was decided was reasonable behaviour. Which really might be just reraising the original exception if there's only one format choice rather than wrapping it in a ValueError.